pkiintergration-fe/src/modules/pki-managed-certificates/pki-managed-certificates.component.ts

```diff
@@ -2,19 +2,23 @@ import { Component, OnInit } from '@angular/core';
 import { BsModalService } from 'ngx-bootstrap/modal';
 import { FetchClient } from '@c8y/client';
 import { CreateCertificateComponent } from './create-certificate/create-certificate.component';
+import { AlertService } from '@c8y/ngx-components';
+import { CreateCertificateService } from '@services/create-certificate.service';
 @Component({
   selector: 'app-pki-managed-certificates',
   templateUrl: './pki-managed-certificates.component.html'
 })
 export class PkiManagedCertificatesComponent {
   data: any;
```

Comment:
issue: set a proper type definition

`any` is a no-go. if it's a rare use-case where `any` must be used, document the reasoning.